allennlp/data/token_indexers/token_indexer.py

```diff
@@ -59,11 +59,13 @@ def pad_token_sequence(self,
                            desired_num_tokens: int,
                            padding_lengths: Dict[str, int]) -> List[TokenType]:
         """
-        This method pads a list of tokens to ``desired_num_tokens``, including any necessary
-        internal padding using whatever lengths are relevant in ``padding_lengths``, returning a
-        padded copy of the input list.  If each token is a single ID, this just adds 0 to the
-        sequence (or truncates the sequence, if necessary).  If each token is, e.g., a list of
-        characters, this method will pad both the characters and the number of tokens.
+        This method pads a list of tokens to ``desired_num_tokens`` and returns a padded copy of the
+        input text.  If the input token list is longer than ``desired_num_tokens`` then it will be
```

Comment:
Instead of "input text", I'd use "input list", "input tokens", "input token list", or something similar.